controllers/admin/AdminShopGroupController.php

```diff
@@ -137,7 +137,7 @@ public function initContent()
         $this->context->smarty->assign([
             'toolbar_scroll' => 1,
             'toolbar_btn' => $this->toolbar_btn,
-            'title' => $this->toolbar_title,
+            'title' => end($this->toolbar_title),
```

Comment:
@Codencode this shouldn't be required to change

Reply:
If I remove this change I always get the warning.

Reply:
I'll get back to you

Reply:
> I'll get back to you


https://github.com/PrestaShop/PrestaShop/assets/2463965/d51afb9c-95d6-417b-8a99-bed244e402b9


https://github.com/PrestaShop/PrestaShop/assets/2463965/6e8e8389-f26a-43de-bbb0-cdbba2969ad7

Reply:
@Codencode yep, I reproduce this issue, @jolelievre suggested a new way of registering `|end` modifier that should solve this issue

Reply:
Ok, I'm waiting for @jolelievre to confirm the changes to proceed.

Thanks!

Reply:
And about `implode` and other methods I know it's not related to your bug exactly and it doesn't help to fix it, but we're trying to anticipate similar bugs that could occur because of missing modifiers (that ere removed by smarty). So while we're modifying the modifiers we can add the ones I suggested after searching into all the smarty templates from the core. Hopefully, we didn't miss too many